pkg/trace/operation.go

```diff
@@ -199,6 +199,20 @@ func (o *Operation) Panic(args ...interface{}) {
 	}
 }
 
+func (o *Operation) Fatalf(format string, args ...interface{}) {
+	o.Fatal(fmt.Sprintf(format, args...))
+}
+
+func (o *Operation) Fatal(args ...interface{}) {
+	msg := fmt.Sprint(args...)
+
+	Logger.Fatalf("%s: %s", o.header(), msg)
```

Comment:
I think this will just exit the process after logging - so will never get to the operation specific logger code. Same deal for the panic call above.

We should call Error on one of the loggers, then Fatalf on the other (probably on the global as that's assured to be there, but it means reordering the calls). Same for panic.

Reply:
I think the better fix here is to just avoid using the Fatal or Panic log methods. I think they're generally ill-advised, but they're especially bad in the context of a server.

That said, I'm not sure what makes the most sense to do 1.3.

Reply:
Thinking about this more, I'm inclined to write this off for the purposes of this PR as "no worse than today".